### PR TITLE
Send email individually; Clear curatorsToNotify after sending email;

### PR DESCRIPTION
**Pull request by zhx828**

### CODE REVIEW

File: web/yo/app/views/curationQueue.html [hunk 1]
```diff
@@ -102,26 +102,23 @@ <h3 class="headerStyle">Curation Queue
...
+                        ng-options="m for m in data.curatorNotificationList"
+                        ng-change="prepareSendEmail()">
                 </select>
-            </div>
-            <div class="col-sm-2">
-                <button tyle="button" class="btn btn-success"
-                        ng-disabled="!curatorsToNotify || curatorsToNotify.length === 0 || email.status.sending || email.returnMessage"
-                        ng-click="sendEmail(curatorsToNotify)">
+                <button type="button" class="btn btn-sm btn-success"
+                        ng-disabled="!data.curatorsToNotify || data.curatorsToNotify.length === 0 || email.status.sending || email.returnMessage"
```
Comment (JiaoJiao123) [inline web/yo/app/views/curationQueue.html]: we should remove email.returnMessage in the or statement. It never get executed.

File: web/yo/app/views/curationQueue.html [hunk 2]
```diff
@@ -102,26 +102,23 @@ <h3 class="headerStyle">Curation Queue
...
+                        ng-click="sendEmail(data.curatorsToNotify)">
```
Comment (JiaoJiao123) [inline web/yo/app/views/curationQueue.html]: remove parameter from the sendEmail()

File: web/yo/app/views/curationQueue.html (hunk 1 above)
Comment (zhx828) [reply web/yo/app/views/curationQueue.html]: Done

File: web/yo/app/views/curationQueue.html (hunk 2 above)
Comment (zhx828) [reply web/yo/app/views/curationQueue.html]: Done

**Pull request merged**